azure-ts-appservice-springboot/infrastructure/Pulumi.yaml

```diff
@@ -0,0 +1,3 @@
+name: jenkins-tutorial
+runtime: nodejs
+description: A minimal Azure TypeScript Pulumi program
```

Comment:
Consider changing the description from the default

Reply:
Done